libkbfs/backpressure_disk_limiter.go

```diff
@@ -102,22 +102,22 @@ func (bt backpressureTracker) currLimit() float64 {
 	return math.Min(limit, float64(bt.limit))
 }
 
-func (bt backpressureTracker) freeFrac() float64 {
+func (bt backpressureTracker) usedFrac() float64 {
```

Comment:
I have no idea / forgot why this was named `freeFrac`, since it's used/something, so I changed it to a better name.